skietbaan_fe/src/components/RegisterMemberStyles.css

```diff
@@ -1,26 +1,22 @@
 @media all and (min-width: 480px) {
-    .Register {
+    .RegisterMember {
       padding: 60px 0;
     }
-  
-    .Register form {
+    .RegisterMember form {
       margin: 0 auto;
       max-width: 320px;
     }
   }
-  
   body{
     background-color:red;
   }
-  
   .test{
     width:50%
   }
   .col-1,.col-2,.col-3,.col-4,.col-5,.col-6,.col-7,.col-8,.col-9,.col-10,.col-11,.col-12{
     float:left;
-  
   }
-  .centre-register{
+  .centre-registermember{
     width:94%;
```

Comment:
[disapprove]

Reply:
any suggestions?

Reply:
@DinolanNaidoo firstly.....if you are using hyphens to separate words, don't go camel case mid way through
.center-register-member

and secondly, why are you only extending the width 94%?  very...very.....very rarely do you do something like that. What is the desired effect you wish to achieve with this? send me some screenshots if you can please